crates/bevy_ui/src/lib.rs

```diff
@@ -39,9 +41,33 @@ pub enum UiSystem {
     Focus,
 }
 
+#[derive(Debug)]
+/// The current scale of the UI for all windows
```

Comment:
Is there a conceptual reason this should be global instead of per-UI camera?

For example, I've needed a custom scale for different `vscode` windows because of different monitor scaling requirements - e.g. for presenting on one with notes on the other. Although I appreciate that is slightly different because those windows are different processes.

Reply:
Not a conceptual one no. So far ui-handling is not done per-window though, so I didn't go as far as understand how multiple window bevy works technically.